runtime/queries/fortran/highlights.scm

```diff
@@ -0,0 +1,170 @@
+(identifier) @variable
```

Comment:
It looks like these queries were written for nvim-treesitter. nvim-treesitter has reversed precedence so this has the lowest priority for nvim but everything else that uses tree-sitter uses the reverse: this is the highest precedence stanza in helix, so the `@variable.parameter` capture on L136 can't match. To fix it you can move this line down below any other line that captures the `(identifier)` node.